packages/cli/src/handlers/dbt/compile.ts

```diff
@@ -126,7 +126,7 @@ async function dbtList(options: DbtCompileOptions): Promise<string[]> {
             'unique_id',
         ];
         const version = await getDbtVersion();
-        // only dbt 1.5 and above support --quiet flag
+        // only dbt 1.7 and above support --quiet flag
```

Comment:
thoughts on this comment - true that >=1.5 support the `quiet` flag but this PR drops support of it.